README.md

```diff
@@ -15,6 +15,35 @@ At installation make sure that you choose the following folder to store data: `/
 
 Since this folder is created only in the `tao` image, it won't be accessible by the `web` container, which is good for security reasons.
 
+
+You can execute this docker image building (specified in next sections) and run the next command:
```

Comment:
Varios cambios:

- Luego del título de USAGE pongamos este texto:

The image is published in docker hub at [devsu/tao](https://hub.docker.com/r/devsu/tao).

- No es que va a ser necesario construir para usar. Deben poder usar la imagen que estaría subida en el docker-hub. Dado que el usage ya contempla la parte del docker compose, pongamos así el texto:

If you don't want to use docker-compose, you can also install and run TAO using the following command.

```
docker run --env DB_HOST=https://example.org --env DB_NAME=myDB --env DB_USER=myDBUser --env DB_PASSWORD=myDBPass --env USER=myTaoAdminUser --env PASSWORD=myTenLengthAlfanumericTaoAdminPassword devsu/tao
```

(Notar al final del comando que no es `tao` sino `devsu/tao`).

Reply:
Corregido de acuerdo a tus especificaciones.